DashWallet/Sources/UI/CrowdNode/CrowdNodeModel.swift

```diff
@@ -279,4 +289,29 @@ extension CrowdNodeModel {
         crowdNode.stopTrackingLinked()
         WKWebView.cleanCrowdNodeCache()
     }
+    
+    func signAndSendEmail(email: String) async throws -> Bool {
+        guard !crowdNode.accountAddress.isEmpty else { return false }
+        emailForAccount = email
+        
+        let wallet = DWEnvironment.sharedInstance().currentWallet
+        let result = await wallet.seed(withPrompt: NSLocalizedString("Sign the message", comment: "CrowdNode"), forAmount: 1)
+            
+        if !result.1 {
+            let key = wallet.privateKey(forAddress: crowdNode.accountAddress, fromSeed: result.0!)
+            let signResult = await key?.signMessageDigest(email.magicDigest())
+                
+            if signResult?.0 == true {
+                let signature = (signResult!.1 as NSData).base64String()
+                try await crowdNode.registerEmailForAccount(email: email, signature: signature)
+                return true
+            }
+        }
+        
+        return false
+    }
```

Comment:
We sign the email that we send with a private key of our account address - that's how CrowdNode links those two together.